src/main/java/com/aws/greengrass/builtin/services/lifecycle/LifecycleIPCEventStreamAgent.java

```diff
@@ -370,7 +376,7 @@ public PauseComponentResponse handleRequest(PauseComponentRequest request) {
                 if (component instanceof GenericExternalService) {
```

Comment:
this check includes lambdas, are we good with that?

Reply:
Yes